src/cli/backup/exchange_e2e_test.go

```diff
@@ -117,7 +117,7 @@ func (suite *ExchangeCMDWithFlagsE2ESuite) TestBackupCreateExchange_azureIDFromC
 		"--config-file", suite.cfgFP)
 	cli.BuildCommandTree(cmd)
 
-	cmd.SetErr(&suite.recorder)
+	cmd.SetOut(&suite.recorder)
```

Comment:
Did we change where we print out the output to? Do you know what changed caused this nightly test to fail? I just wanted to make sure we did not accidentally change the behaviour of the print.
/cc @ryanfkeepers

Reply:
These were newly added test cases. Implementation was not correct. We haven't changes any behaviour of the print.